package.json

```diff
@@ -1,13 +1,15 @@
 {
   "name": "angular-pipes",
-  "version": "9.1.0",
+  "version": "9.2.0",
```

Comment:
Would probably be better to change this to 10.0.0 because of the size of the changes.